tests/components/test_component_delaunay.cpp

```diff
@@ -82,14 +83,15 @@ TEST_CASE("delaunay_2d_random", "[components][delaunay]")
     Eigen::MatrixXd vertices;
     Eigen::MatrixXi faces;
```

Comment:
for future reference, could have used structured bindings for these to clean the notation up

Reply:
I did, didn't I? 
I just couldn't use the `auto [vertices, faces]` notation because the variable declaration would have been wrapped inside the `CHECK`.